components/ui/themes/PurpleTheme.ts

```diff
@@ -753,6 +753,11 @@ export const PurpleTheme = {
       borderTopLeftRadius: 0,
       borderTopRightRadius: 0,
     },
+    halfButton: {
```

Comment:
Do we need this half button styles anymore ?

Reply:
Yes, this is being used in DualMessageOverlay atm